modules/multi-architecture-scheduling-examples.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assembly 
+//
+//post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc
+
+:_content-type: CONCEPT
+[id="multi-architecture-scheduling-examples_{context}"]
+
+= Sample multi-architecture node workload deployments 
+
+There are various cases to consider when you are scheduling workloads on a cluster with compute nodes of different architectures. 
+
+Using node affinity to schedule workloads on a node:: To allow a workload to be scheduled on only a set of nodes with architectures supported by its images, you can set the `spec.affinity.nodeAffinity` field in your pod template specification.
++
+.Example deployment with the `nodeAffinity` set to certain architectures 
+[source,yaml]
```

Comment:
See YAML guidance on https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting-for-kubernetes-and-openshift-api-objects

![Screenshot from 2023-09-25 17-09-16](https://github.com/openshift/openshift-docs/assets/57954076/c6896b67-9592-46e4-a86f-2a42211595dc)